tests/generate_packageTest.py

```diff
@@ -83,6 +83,41 @@ def test_command(self, call_mock):
         call_mock.assert_called_with(['twine', 'upload', './dist/*'])
 
 
+class MainSingleTest(unittest.TestCase):
```

Comment:
good tests but a bit incomplete. we should have tests for all situations. for example, 2 or 3 bears given as "-b A B C D".
also a test for no matching bears (incorrect names).